frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTable/helpers.ts

```diff
@@ -135,23 +127,39 @@ export type ISoftwareVulnFiltersParams = {
   maxCvssScore?: number;
 };
 
-export const getSoftwareVulnFiltersForQueryKey = (
-  vulnFilters: ISoftwareVulnFilters
+const isValidNumber = (
+  value: any,
+  min?: number,
+  max?: number
+): value is number => {
+  // Check if the value is a number and not NaN
+  const isNumber = typeof value === "number" && !isNaN(value);
+
+  // If min or max is provided, check if the number is within the range
+  const withinRange =
+    (min === undefined || value >= min) && (max === undefined || value <= max);
+
+  return isNumber && withinRange;
```

Comment:
added extra check for min/max range in case a user enters a custom min max (api allows this)